core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowResource.scala

```diff
@@ -242,17 +243,12 @@ class WorkflowResource {
   @GET
   @Path("/workflow-ids")
```

Comment:
can we change this endpoint url to be more meaningful?

Reply:
Renamed.